include/deal.II/cgal/intersections.h

```diff
@@ -61,20 +61,15 @@ namespace CGALWrappers
 
   /**
    * Same function as above, but working directly with vertices.
+   * @note the vertices have to be given in CGAL order
```

Comment:
If you could document what `dim0` etc. do that would help a lot too.

Reply:
Sure, I can do that.